core/build-readme.js

```diff
@@ -80,7 +80,7 @@ const withExternalIcon = $ => {
 }
 
 const createWithRelativeLinks = ({ isMarkdownPath }) => ($, { ref }) => {
-  forEach(TAGS, (htmlTags, propName) => {
+  Array.from(TAGS).forEach((htmlTags, propName) => {
```

Comment:
is this introducing a bug?

```
> require('html-urls').TAGS
{ background: [ 'body' ],
  cite: [ 'blockquote', 'del', 'ins', 'q' ],
  data: [ 'object' ],
  href: [ 'a', 'area', 'embed', 'link' ],
  icon: [ 'command' ],
  longdesc: [ 'frame', 'iframe' ],
  manifest: [ 'html' ],
  poster: [ 'video' ],
  pluginspage: [ 'embed' ],
  pluginurl: [ 'embed' ],
  src:
   [ 'audio',
     'embed',
     'frame',
     'iframe',
     'img',
     'input',
     'script',
     'source',
     'track',
     'video' ] }
> Array.from(require('html-urls').TAGS)
[]
```

Reply:
I'm not sure, let's just use the lodash implementation - that's the safest option.